recipes/quickfast/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.8)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(test ${PROJECT_SOURCE_DIR}/main.cpp)
+
+target_link_libraries(test ${CONAN_LIBS})
+
+target_compile_features(test PUBLIC cxx_std_11)
```

You can lower the required cmake version by doing:
```suggestion
set_property(TARGET test PROPERTY CXX_STANDARD 11)
```